Backpack-SwiftUI/Badge/Classes/BPKBadge.swift

```diff
@@ -67,8 +65,35 @@ public struct BPKBadge: View {
             BPKText(title, style: .caption)
                 .foregroundColor(style.foregroundColor)
         }
-        // Align text and icon frame.
-        .frame(minHeight: BPKSpacing.base.value)
+    }
+}
+
+// Helper functions to apply the outline style
+fileprivate struct Outline: ViewModifier {
+    let color: BPKColor
+    let cornerRadius: BPKCornerRadius
+    
+    func body(content: Content) -> some View {
+        return content
+            .overlay(
+                RoundedRectangle(cornerRadius: cornerRadius)
+                    .stroke(Color(color))
+            )
+    }
+}
+
+fileprivate extension View {
+    func outline(_ color: BPKColor?, cornerRadius: BPKCornerRadius) -> some View {
+        guard let color = color else {
+            return AnyView(self)
+        }
+
+        return AnyView(
+            ModifiedContent(
+                content: self,
+                modifier: Outline(color: color, cornerRadius: cornerRadius)
+            )
```

Comment:
I think this can be a @ViewBuilder
```swift
@ViewBuilder
func outline(_ color: BPKColor?, cornerRadius: BPKCornerRadius) -> some View {
  if let color = color {
     self
  } else {
     ModifiedContent(..)
  }
}
```